portfolio/src/main/webapp/index.html

```diff
@@ -42,6 +42,9 @@ <h1 style=font-size:40px class="heading-font">Apoorva Dhanala's Portfolio</h1>
       <h3 class="links-font"><a href="resume.html">MY RESUME</a></h3>
       <h3 class="links-font"><a href="photography.html">MY PHOTOGRAPHY</a></h3>
       <h3 class="links-font"><a href="favorites.html">FAVORITE SONGS, TV SHOWS, AND MORE!</a></h3>
+      <h3>Click here to see a random quote:</h3>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Hi!</button>
```

Comment:
This should just be named getRandomQuote() still. UsingArrowFunctions is specifically describing an implementation detail (specifically, that this is using the arrow syntactic sugar to create an anonymous function).

Reply:
An alternative might be updateQuote() (to reflect the fact that it's pulling a quote from the server and updating it)

Reply:
Oh okay that makes sense! I'll change it now